packages/components/src/ellipsis-menu/index.js

```diff
@@ -33,8 +33,12 @@ class EllipsisMenu extends Component {
 			);
 		};
 
-		const renderContent = () => (
-			<NavigableMenu className="woocommerce-ellipsis-menu__content">{ children }</NavigableMenu>
+		// @todo Make all children rendered by render props so Dropdown args can be passed?
```

Comment:
`EllipsisMenu` is essentially a wrapper for `Dropdown`, but by rendering children instead of using render props [like `Dropdown` does](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/dropdown/index.js#L94), we're removing built in functionality. In this case, use of argument `onToggle` so that dropdown contents can have access to this argument.

In a follow up PR, I'd like to change `EllipsisMenu` such that it accepts a `renderContent` prop instead of children.